transforms/PathologyLabelSoften.py

```diff
@@ -38,19 +40,19 @@ def __call__(self, label: NdarrayOrTensor, meta_dict: Dict) -> Tuple[NdarrayOrTe
         label, prev_type, device = convert_data_type(label, np.ndarray)
         # labels are pseudo-3d, means format is C [[H, W], 1]
         # squeezing makes working with labels easier
-        label = label.squeeze(-1)
-        logger.debug("[PathologyLabelSoften]: label shape after squeeze:", label.shape)
```

Comment:
I found the logger useful for debugging. I think the transform is not perfect yet, so maybe leave the logger? It won't print during training unless explicitly asked to.